src/markdown/symbols/mdast-extension-symbols.js

```diff
@@ -0,0 +1,27 @@
+export const fromMarkdown = {
```

Comment:
I agree with this in general, and moved the functions accordingly in most places. The one exception where I feel this isn't a good idea is for the `tokenize` functions in micromark extensions. Since those functions describe transitions of a state machine, it makes sense to me that they're written in order of when those states are reached.